scripts/generate-config.js

```diff
@@ -98,26 +98,7 @@ commands:
             imageName:
                 type: string
                 description: Cypress base docker image to test
-            checkNodeVersion:
```

Comment:
I'm assuming all these checkNodeVersions are unnecessary now that we are using these generator scripts?